UI/cypress/e2e/shortlist.js

```diff
@@ -144,3 +143,39 @@ When(
     cy.wait(500);
   }
 );
+
+Then("total amount of shortlisted TPs is {int}", (expectedTotal) => {
```

Comment:
Isn't this the same as `"the shortlist shows as having {int} entries on the results page"` above?